tests/src/test/scala-2.x/cats/tests/FunctionKLiftSuite.scala

```diff
@@ -0,0 +1,50 @@
+package cats.tests
+
+import cats.data.NonEmptyList
+import cats.arrow.FunctionK
+import cats.implicits._
+import org.scalacheck.Prop._
+import cats.laws.discipline.arbitrary._
+
+class FunctionKLiftSuite extends CatsSuite {
+
+  test("lift simple unary") {
+    def optionToList[A](option: Option[A]): List[A] = option.toList
+    val fOptionToList = FunctionK.lift(optionToList _)
+    forAll { (a: Option[Int]) =>
+      assert(fOptionToList(a) === (optionToList(a)))
+    }
+
+    val fO2I: FunctionK[Option, Iterable] = FunctionK.lift(Option.option2Iterable _)
+    forAll { (a: Option[String]) =>
+      assert(fO2I(a).toList === (Option.option2Iterable(a).toList))
+    }
+
+    val fNelFromListUnsafe = FunctionK.lift(NonEmptyList.fromListUnsafe _)
+    forAll { (a: NonEmptyList[Int]) =>
+      assert(fNelFromListUnsafe(a.toList) === (NonEmptyList.fromListUnsafe(a.toList)))
+    }
+  }
+
+  test("hygiene") {
+    trait FunctionK
+    def optionToList[A](option: Option[A]): List[A] = option.toList
+    val fOptionToList = cats.arrow.FunctionK.lift(optionToList _)
+    forAll { (a: Option[Int]) =>
+      assert(fOptionToList(a) === (optionToList(a)))
+    }
+  }
```

Comment:
@larsrh this trait FunctionK is not used here. I wonder is left here intentionally or just an oversight?

Reply:
That's on purpose, since it is a hygiene test. We're checking whether the macro uses the correct `FunctionK` trait.